ludwig/backend/ray.py

```diff
@@ -46,6 +46,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Disable placement groups on dask
+dask.config.set(annotations={"ray_remote_args": {"placement_group": None}})
```

Comment:
I believe this wasn't working before, is that not the case?

Reply:
Removed, yet to test out the alternative fix.

Reply:
Confirmed.